### PR TITLE
Glyph Title Transform Function for CSS styles

### PR DESCRIPTION
**Pull request by ScaryDonetskiy**
If you need some transform in titles of CSS classes for your icons, you can use option glyphTransformFn with glyphs metadata object

### CODE REVIEW

File: src/__tests__/standalone-test.js [hunk 1]
```diff
@@ -223,3 +223,27 @@ test('should throw error of config file not found', (t) => {
...
+        glyphTransformFn: (obj) => {
+            obj.name += '_transform';
+        },
+        template: 'css'
+    }).then((result) => {
+        t.true(typeof result.svg === 'undefined');
+        t.true(typeof result.ttf === 'undefined');
+        t.true(typeof result.woff === 'undefined');
+        t.true(typeof result.woff2 === 'undefined');
+        t.true(isEot(result.eot));
```
Comment (evilebottnawi) [inline src/__tests__/standalone-test.js]: Here I think it is not necessary to check the fonts, as is done in other tests, let's just leave the test classes

**Comment by evilebottnawi** [comment]
@ScaryDonetskiy forgotten, need add documentation in `README.md` this option.

**Comment by ScaryDonetskiy** [comment]
@evilebottnawi Ok, I fixed it

**Comment by evilebottnawi** [comment]
@ScaryDonetskiy let's wait CI green and merge :+1:

**Comment by ScaryDonetskiy** [comment]
@evilebottnawi Could you update npm package version after merge?

**Comment by evilebottnawi** [comment]
@ScaryDonetskiy yep

**Pull request merged**